packages/inference/src/InferenceClient.ts

```diff
@@ -34,40 +32,33 @@ export class InferenceClient {
 				enumerable: false,
 				value: (params: RequestArgs, options: Options) =>
 					// eslint-disable-next-line @typescript-eslint/no-explicit-any
-					fn({ ...params, accessToken } as any, { ...defaultOptions, ...options }),
+					fn({ endpointUrl: defaultOptions.endpointUrl, accessToken, ...params } as any, {
```

Comment:
i switched the order here btw @coyotte508 so that method params take precedence over the instance's props, which is what we always want i assume?

Reply:
sure